projects/example-app/src/app/books/containers/collection-page.component.ts

```diff
@@ -34,11 +32,9 @@ import * as fromBooks from '@example-app/books/reducers';
   ],
 })
 export class CollectionPageComponent implements OnInit {
-  books$: Observable<Book[]>;
+  books = this.store.selectSignal(fromBooks.selectBookCollection);
```

Comment:
I'm fine with `example-app` changes, but it's maybe better to keep the observable implementation at least until signals become stable.

Reply:
Agreed. It was mainly for testing in a few areas